Backend/BikesRentalServer.Services/MalfunctionsService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using BikesRentalServer.Infrastructure;
+using BikesRentalServer.Models;
+using BikesRentalServer.Services.Abstract;
+using BikesRentalServer.DataAccess.Repositories.Abstract;
+
+namespace BikesRentalServer.Services
+{
+    public class MalfunctionsService : IMalfunctionsService
+    {
+        private readonly UserContext _userContext;
+        private readonly IMalfunctionsRepository _malfunctionRepository;
+        private readonly IBikesRepository _bikesRepository;
+        private readonly IUsersRepository _usersRepository;
+        public MalfunctionsService(UserContext userContext, IMalfunctionsRepository malfunctionRepository, IBikesRepository bikesRepository, IUsersRepository usersRepository)
```

Comment:
Można się pokusić o rozbicie tego na 4 linie, bo długie się robi (nie wiem jak to w IDE wygląda, może nie ma potrzeby)